### PR TITLE
feat: update round details design

### LINKED ISSUE #198: Update Round Details design
**User Story:**
As a round operator
I want to see the high-fidelity design for Round Manager
So that I can trust the dApp

**Acceptance Criteria**
GIVEN that I am on the Round Details page
WHEN I haven't received any applications
THEN I see the empty state

GIVEN that I am on the Round Details page
WHEN I have received applications
THEN I see the populated state

NOTE: you do not need to add in this copy / button —> that will come in a later feature
image

**Product & Design Links:**
[Empty state](https://www.figma.com/file/K31Tf7tSn10BuN65VC6YLc/Round-Manager---Hi-Fidelity-Designs?node-id=92%3A1866)
[Populated state](https://www.figma.com/file/K31Tf7tSn10BuN65VC6YLc/Round-Manager---Hi-Fidelity-Designs?node-id=101%3A1450)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary
- Showing the project banner + logo in application cards will be implemented in #249

### PR DESCRIPTION
**Pull request by shavinac**
##### Description

* updates round details page to match high fidelity mocks

##### Refers/Fixes

resolves #198

##### Testing
round details with no applications
round details with 1 approved application

### CODE REVIEW

File: packages/round-manager/src/test-utils.tsx [hunk 1]
```diff
@@ -68,3 +66,39 @@ export const makeStubRound = (overrides: Partial<Round> = {}): Round => {
...
+    projectsMetaPtr: {
+      protocol: faker.datatype.number(),
+      pointer: faker.datatype.uuid()
+    },
+    ...overrides
+  }
+}
+
+export const renderWrapped = (ui: JSX.Element) => {
+  render(
```
Comment (shavinac) [inline packages/round-manager/src/test-utils.tsx:97]: `render` is already imported as `rtlRender`, we should pick one or the other
```suggestion
  rtlRender(
```

**Review by chibie — CHANGES_REQUESTED** [review]
can you update the card design for the applications? (including banner + logo)

Screenshot 2022-08-08 at 22 58 39

**Comment by shavinac** [comment]
> can you update the card design for the applications? (including banner + logo)

i thought adding the project banner/logo would be covered in https://github.com/gitcoinco/grants-round/issues/249

**Comment by chibie** [comment]
> > can you update the card design for the applications? (including banner + logo)
> 
> i thought adding the project banner/logo would be covered in #249

oh yes! my bad! you're right

**Review by chibie — APPROVED** [review]
neat! 🚀

**Pull request merged**